niceday-broker/.env-example

```diff
@@ -1,2 +1,3 @@
-THERAPIST_USER_ID = 38714
-NICEDAY_TOKEN = asldkfjasdljr12321ljafd
\ No newline at end of file
+THERAPIST_USER_ID=38714
+NICEDAY_TOKEN=asldkfjasdljr12321ljafd
+RASA_AGENT_URL=http://localhost:5005/webhooks/rest/webhook
```

Comment:
Yes, it is there twice because the niceday-broker one is specific for the niceday-broker. For now they are identical because the 'root' one only needs secrets for the broker.